packages/clay-empty-state/src/EmptyState.tsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * © 2020 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import React from 'react';
+
+interface IProps extends React.HTMLAttributes<HTMLDivElement> {
```

Comment:
Instead of extending all attributes from the div element here, lets move these extra html attribute props to its own prop, `containerProps`. This would be similar to Checkbox, Radio and Toggle. 

Example:
```ts
interface IProps {
	containerProps?: React.HTMLAttributes<HTMLDivElement>
	// ...
}

const ClayEmptyState: React.FunctionComponent<IProps> = ({
	containerProps = {},
	//...

	const {className, ...otherContainerProps} = containerProps;

	return <div className={classNames(className, ...)} {...otherContainerProps}
```

Reason I recommend `containerProps` instead of just extending is because we would have a conflict with `title` and `containerProps.title`. I think it'll be more common for people to use this component with only title, description, and imageSrc.